index.js

```diff
@@ -106,6 +106,14 @@ Sonus.start = sonus => {
   sonus.mic.pipe(sonus.detector)
 }
 
+Sonus.pause = sonus => {
```

Comment:
More terse
```javascript Sonus.pause = sonus => sonus.mic.pause()```
```javascript Sonus.resume = sonus => sonus.mic.resume()```


Also, do you want error handling for if its already paused / active?